pages/registries.py

```diff
@@ -15,22 +17,35 @@
 
 
 class BaseRegistriesPage(OSFBasePage):
-
-    # Components
+    base_url = settings.OSF_HOME + '/registries/'
```

Comment:
Nitpick: since you're already importing `urljoin` here, you should use that.

Reply:
Good catch. Fixed.